packages/fxa-settings/src/models/integrations/oauth-web-integration.ts

```diff
@@ -267,18 +254,6 @@ export class OAuthIntegration extends BaseIntegration<OAuthIntegrationFeatures>
   }
 
   getServiceName() {
-    const permissions = this.getPermissions();
-    // As a special case for UX purposes, any client requesting access to
-    // the user's sync data must have a display name of "Firefox Sync".
-    // This is also used to check against `integration.isSync()`.
-    if (
-      permissions.some((permission) =>
-        permission.includes(Constants.OAUTH_OLDSYNC_SCOPE)
-      )
-    ) {
-      return Constants.RELIER_SYNC_SERVICE_NAME;
```

Comment:
Looking at this again and seeing [this PR description](https://github.com/mozilla/fxa/pull/3770), I think I'll revert these lines from being removed. I don't know what oauth clients ever request Sync scope, but feels safer to just leave it?

Reply:
If you got to the admin panel, it shows the allowed scopes for each RP. There's quite a few that have 'oldsync' as a possible allowed scope.

Reply:
Ah thanks, I haven't checked admin-panel in prod in a bit and forgot I could go there to see. I'll just leave it as-is for now.